api.py

```diff
@@ -8,25 +8,47 @@
 from resources.Signup import Signup
 from resources.Links import Links
 
+from db import db
+
 app = Flask(__name__)
 api = Api(app)
 swagger = Swagger(app)
-
-app.debug = True
-app.config['SECRET_KEY'] = 'super-secret'
-# Setup the Flask-JWT-Extended extension
-app.config['JWT_SECRET_KEY'] = 'super-secret'  # Change this!
 jwt = JWTManager(app)
 
 
 # Load config file
 with open('config.json', mode='r') as config_file:
     CONFIG = json.load(config_file)
 
+# Database connection
+database_username = CONFIG['database']['username']
```

Comment:
I would suggest to change it like:
```suggestion
database_username = CONFIG.get('database', {}).get('username')
```
Otherwise, You will face `KeyError` if config file doesn't have a `database` key.
Please change the other variables as well.